geonode_mapstore_client/client/js/utils/ResourceUtils.js

```diff
@@ -100,6 +106,21 @@ const datasetAttributeSetToFields = ({ attribute_set: attributeSet = [] }) => {
         });
 };
 
+export const getDimensions = ({links, has_time: hasTime} = {}) => {
+    const { url: wmsUrl } = links.find(({ link_type: linkType }) => linkType === 'OGC:WMS') || {};
+    const { url: wmtsUrl } = links.find(({ link_type: linkType }) => linkType === 'OGC:WMTS') || {};
+    const dimensions = [
+        ...(hasTime ? [{
+            name: 'time',
+            source: {
+                type: 'multidim-extension',
+                url: wmtsUrl || (wmsUrl || '').split('/geoserver/')[0] + '/geoserver/gwc/service/wmts'
+            }
+        }] : [])
+    ];
+    return dimensions;
+};
```

Comment:
please add some testing at least for the utility function, thanks

Reply:
Added unit tests